internal/api/attestationconfigapi/cli/main.go

```diff
@@ -66,6 +68,9 @@ func newRootCmd() *cobra.Command {
 	}
 	rootCmd.Flags().StringP("maa-claims-path", "t", "", "File path to a json file containing the MAA claims.")
 	rootCmd.Flags().StringP("upload-date", "d", "", "upload a version with this date as version name.")
+	rootCmd.Flags().BoolP("force", "f", false, "Use force to manually push a new latest version."+
+		" The version gets reported in the cache but the version selection logic is skipped.")
+	rootCmd.Flags().IntP("cache-window-size", "s", 0, "Number of versions to be considered for the latest version.")
```

Comment:
```suggestion
	rootCmd.Flags().IntP("cache-window-size", "s", 15, "Number of versions to be considered for the latest version.")
```

or whatever you want to set as default. Should be a safe default.

Reply:
this is not exported. should we export it?